scripts/zkai_d128_rmsnorm_mlp_fused_gate.py

```
@@ -0,0 +1,394 @@
+        "proof_backend_version": require_str(envelope.get("proof_backend_version"), "proof backend version"),
+        "statement_version": require_str(envelope.get("statement_version"), "statement version"),
+        "statement_commitment": require_str(input_payload.get("statement_commitment"), "statement commitment"),
+        "public_instance_commitment": require_str(input_payload.get("public_instance_commitment"), "public instance commitment"),
+        "non_claims": input_payload.get("non_claims"),
+        "validation_commands": input_payload.get("validation_commands"),
+        "source_artifacts": EXPECTED_PATHS,
+    }
+    return payload
+
+
+def validate_payload(payload: dict[str, Any]) -> None:
```

Ruff (0.15.12)

[warning] Too many branches (13 > 12)

(PLR0912)